### ISSUE TITLE
Unable to AirDrop docset

### ISSUE THREAD

**Issue by ClementPadovani** [Kapeli/Dash-iOS#60]
When using AirDrop to send a docset. 

Dash isn’t recognized as being able to handle docsets, thus iOS is offering the option to save the file to iCloud Drive or to search an app on the App Store.

![screenshot](https://user-images.githubusercontent.com/676023/30776826-0b8769bc-a0ae-11e7-966d-b90933facbb1.png)

**Comment by dcastil** [comment]
I have the same problem with iOS 13.1.3 and Dash 1.8.13.

**Comment by nevillco** [comment]
Also seeing this with iOS 13.1.3 and Dash 1.8.13 - is anyone aware of a workaround?

**Comment by MikePolinske** [comment]
I was able to transfer the generated docset with iMazing.  I tried to use the Finder in macOS Catalina and it just wouldn't work.  I also tried with the file uploaded to my iCloud Drive and the Files app, but Dash is not in the share sheet in iOS 13.3.

In iMazing for my iPhone, I created a new shortcut for Dash, opened that and then drag-and-dropped the Apple API Reference to the iMazing   I did the same thing with the Java 12 docset.
